Samples/src/FSharpWithQSharp/QuantumCode/Operations.qs

```diff
@@ -0,0 +1,77 @@
+﻿namespace QuantumCode
+{
+    open Microsoft.Quantum.Primitive;
+    open Microsoft.Quantum.Canon;
+
+    // The oracle: f(x₀, …, xₙ₋₁) = Σᵢ (rᵢ xᵢ + (1 - rᵢ)(1 - xᵢ)) modulo 2 for a given bit vector r = (r₀, …, rₙ₋₁).
+    // Inputs:
+    //      1) N qubits in arbitrary state |x⟩ (input register)
+    //      2) a qubit in arbitrary state |y⟩ (output qubit)
+    //      3) a bit vector of length N represented as Int[]
+    // The qubit array and the bit vector have the same length.
+    operation Oracle_ProductWithNegationFunction (x : Qubit[], y : Qubit, r : Int[]) : Unit {
```

Comment:
```suggestion
    operation ApplyProductWithNegationFunction (x : Qubit[], y : Qubit, r : Int[]) : Unit {
```

I'd also suggest that, in keeping with style conventions, qubit arguments follow all classical arguments.